### PR TITLE
Added ROTATION in an area definition

### PR DESCRIPTION
**Pull request by loreclem**
Added ROTATION in an area definition in order to rotate an image after resampling.

### CODE REVIEW

File: pyresample/geometry.py [hunk 1]
```diff
@@ -991,7 +997,7 @@ def get_proj_coords_dask(self, blocksize, dtype=None):
     def get_proj_coords_array(self, data_slice=None, cache=False, dtype=None):
         return np.dstack(self.get_proj_coords(data_slice, cache, dtype))
 
-    def get_proj_coords(self, data_slice=None, cache=False, dtype=None):
+    def get_proj_coords(self, data_slice=None, cache=False, dtype=None, target_area=False):
```
Comment (mraspaud) [inline pyresample/geometry.py]: What is target_area for ?

File: pyresample/geometry.py [hunk 2]
```diff
@@ -810,6 +815,7 @@ def create_areas_def_legacy(self):
         fmt += "\tPCS_DEF:\t{proj_str}\n"
         fmt += "\tXSIZE:\t{x_size}\n"
         fmt += "\tYSIZE:\t{y_size}\n"
+        fmt += "\tROTATION:\t{rotation}\n"
```
Comment (mraspaud) [inline pyresample/geometry.py]: Did you check that it works with yaml area files too ?

File: pyresample/geometry.py [hunk 3]
```diff
@@ -653,6 +653,8 @@ class AreaDefinition(BaseDefinition):
         x dimension in number of pixels
     y_size : int
         y dimension in number of pixels
+    rotation: float
+        rotation in degrees to be applied after resampling (negative is cw)
```
Comment (mraspaud) [inline pyresample/geometry.py]: do we need the `after resampling` part ? I feel it could be confusing

File: pyresample/utils.py [hunk 4]
```diff
@@ -273,6 +277,8 @@ def get_area_def(area_id, area_name, proj_id, proj4_args, x_size, y_size,
         Number of pixel in x dimension
     y_size : int
         Number of pixel in y dimension
+    rotation: float
+        Rotation in degrees to be applied after resampling (negative is cw)
```
Comment (mraspaud) [inline pyresample/utils.py]: Same here, `after resampling`

File: pyresample/geometry.py (hunk 1 above)
Comment (loreclem) [reply pyresample/geometry.py]: It was forgotten, I have removed it

**Comment by mraspaud** [comment]
Lgtm. Anyone has more comments, or should I merge this ? @pnuu @davidh-ssec ?

**Comment by pnuu** [comment]
Go for merge

**Comment by mraspaud** [comment]
Merged, Thanks @loreclem !

**Pull request merged**